raiden-ts/src/raiden.ts

```diff
@@ -152,6 +152,7 @@ export class Raiden {
    * @returns token info
    */
   public getTokenInfo: (
+    this: Raiden,
```

Comment:
Can't we make this a `function` so we have access to `this`? If not, typedoc needs to be updated with the new param

Reply:
First, no, because this isn't a method, but a member function, which is assigned in the constructor instead of lying on the prototype. `this` isn't a normal parameter either, but just a special TS syntax to declare the type of `this` constructor for these free-range functions, they don't get docs because of this.